vscode-car-plugin/src/main/java/org/wso2/maven/CARMojo.java

```diff
@@ -101,8 +101,8 @@ public void execute() throws MojoExecutionException {
                 CAppHandler cAppHandler = new CAppHandler(cAppName);
                 List<ArtifactDependency> dependencies = new ArrayList<>();
 
-                cAppHandler.processConfigArtifactXmlFile(projectBaseDir, archiveDirectory, dependencies);
-                cAppHandler.processRegistryResourceArtifactXmlFile(projectBaseDir, archiveDirectory, dependencies);
+                cAppHandler.processConfigArtifactXmlFile(archiveLocation, archiveDirectory, dependencies);
+                cAppHandler.processRegistryResourceArtifactXmlFile(archiveLocation, archiveDirectory, dependencies);
```

Comment:
When we use the maven resource elements inside the build configuration, all the resources (src/) will end up in **${project.build.directory}/calsses** directory. Therefore, whenever this "**archiveLocation**" is configured with a custom location, the project build will try to access the resources in
**\<archiveLocation\>/classes** directory and will fail with **java.io.FileNotFoundException** as it cannot find the resources there. (As they are now copied to the ${project.build.directory}/calsses directory).

Please refer to the comment and suggestion here. https://github.com/wso2/maven-tools/issues/64#issuecomment-646725394